pkg/volume/azure_dd/azure_common.go

```diff
@@ -93,19 +94,20 @@ func makeDataDisk(volumeName string, podUID types.UID, diskName string, host vol
 		volumeName:      volumeName,
 		diskName:        diskName,
 		podUID:          podUID,
+		plugin:          plugin,
 	}
 }
 
-func getVolumeSource(spec *volume.Spec) (*v1.AzureDiskVolumeSource, error) {
+func getVolumeSource(spec *volume.Spec) (*v1.AzureDiskVolumeSource, bool, error) {
```

Comment:
nit: add documentation of returned fields

Reply:
fixed, add a named parameter is enough, not necessary to add doc